gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
+#Mon Sep 19 11:01:26 CEST 2022
 distributionBase=GRADLE_USER_HOME
-distributionPath=wrapper/dists
 distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-bin.zip
-distributionSha256Sum=f6b8596b10cce501591e92f229816aa4046424f3b24d771751b06779d58c8ec4
```

Comment:
We should revert changes to this file; it has dropped the sha for Gradle installation integrity checks.